packages/seller-client/src/components/BuyerDetails/RatingStars.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import StarIcon from '@material-ui/icons/Star';
+import StarHalfIcon from '@material-ui/icons/StarHalf';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
+
+const STARS = new Array(5).fill(1);
```

Comment:
Filling these boys with null makes it super clear that the array values aren't necessary.

I think I actually prefer using the `new Array(5).fill(null)` or `range` (from lodash) directly since `STARS` doesn't actually contain any stars. Super nit pick tho. No big deal

```suggestion
const STARS = new Array(5).fill(null);
```